geoportailv3/static/js/exclusionmanagerservice.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @fileoverview This file defines the Exclusion service. This service manages
+ * the exclusion between layers.
+ */
+goog.provide('app.ExclusionManager');
+
+goog.require('app');
+goog.require('app.BlankLayer');
+goog.require('app.Notify');
+goog.require('ngeo.BackgroundLayerMgr');
+
+
+
+/**
+ * @param {ngeo.BackgroundLayerMgr} ngeoBackgroundLayerMgr Background layer
+ *     manager.
+ * @param {app.BlankLayer} appBlankLayer Blank layer service.
+ * @param {app.Notify} appNotify Notify service.
+ * @constructor
```

Comment:
`@ngInject` is missing, so I guess the injection does not work in non-debug mode.